src/release.sh

```diff
@@ -4,6 +4,7 @@ set -x
 
 SRC_HASH=`git rev-parse --verify HEAD`
 APP_NAME=`node -e 'console.log(require("./package.json").insights.appname)'`
+NODE_VERSION=`node -e 'console.log(require("./package.json").engines.node)'`
```

Comment:
You might need conditional access. Not every repo has the engines setup.
```
NODE_VERSION=`node -e 'console.log(require("./package.json")?.engines?.node)'`
```

Reply:
Fixed, thanks!